Fitness/Controllers/HomeController.swift

```diff
@@ -185,12 +196,17 @@ extension HomeController: UICollectionViewDataSource {
             let url = URL(string: tags[indexPath.row].imageURL)
             categoryCell.image.kf.setImage(with: url)
             return categoryCell
+        case .pros:
+            let discoverPros = collectionView.dequeueReusableCell(withReuseIdentifier: DiscoverProsCell.identifier, for: indexPath) as! DiscoverProsCell
+            discoverPros.collectionView.dataSource = self
+            discoverPros.collectionView.delegate = self
+            return discoverPros
         }
     }
 
     func numberOfSections(in collectionView: UICollectionView) -> Int {
         if collectionView != mainCollectionView { return 1 }
-        return 3
+        return 4
```

Comment:
This seems odd. If the sections are hardcoded, rather than doing sections.insert multiple times can we just hardcode the nested arrays and return the count of the resulting 2D array?

Reply:
I was going off of previous code, but what you mentioned makes more sense. I'll change it for the next commit